docs/developer-docs/setup/cycles/cycles-ledger.md

```diff
@@ -0,0 +1,158 @@
+
+- Cycles wallets are a complex solution to associate cycles with principal IDs.
+- Cycles wallets consume cycles themselves, meaning developers need to pay cycles to store cycles.
+- Since the cycles wallet canister must be deployed locally, if a cycles wallet canister ID is lost, the cycles in that cycles wallet canister are also lost. 
+
+The cycles ledger simplifies cycle management by providing the ability for principal IDs to hold cycles. 
+
+The cycles ledger complies with the ICRC-2 standard. In addition to standard ledger functionality, the cycles ledger canister also interacts with the NNS canisters and user canisters to provide cycles ledger-specific functions, such as:
+
+- Accepting incoming cycles sent from other canisters.
+- Sending cycles to other canisters.
+- Creating new canisters using cycles.
+
+:::caution
+It is important to note that the cycles ledger does not provide the ability to make arbitrary calls to other canisters with cycles, while the cycles wallet can. This is because open call contexts may cause complications with the cycles ledger. 
+
+If you need to make arbitrary calls to other canisters with cycles, a cycles wallet can be used as before. The cycles ledger can be used to load the cycles wallet with cycles.
+:::
+
+## Obtaining ICP
+
+To use the cycles ledger, first you need ICP tokens. There are a few different ways to obtain ICP tokens, such as:
+
+- Purchasing ICP tokens directly through an exchange that lists ICP tokens available for trade. To see all available places to purchase ICP tokens, check out this [page](https://coinmarketcap.com/currencies/internet-computer/markets/).
+- Trading other tokens for ICP on a decentralized exchange.
+- Receiving tokens as rewards for participating in the governance of the Internet Computer.
+- Receiving a grant of tokens through the Internet Computer Association (ICA) or the DFINITY Foundation.
+- Receiving tokens as remuneration for providing computing capacity as a node provider.
+
+To obtain ICP, you also need to have an account to transfer those tokens into. To get your account ID, first determine which dfx identity you are using with the command:
+
+```
+dfx identity whoami
+```
+
+If you're using the default identity, you should create a new identity with the command:
+
+```
+dfx identity new IDENTITY_NAME
+```
+
+Replace `IDENTITY_NAME` with the an identity name of your choice, such as 'MyIdentity`.
+
+Your identity's seed phrase will be returned. Be sure to save this in a secure location. Then, set this identity to be used by the current session:
+
+```
+dfx identity use IDENTITY_NAME
+```
+
+Then, to get the account ID for this identity, use the command:
+
+```
+dfx ledger account-id
+```
+
+This will display your account number on the ICP ledger. It looks similar to this:
+
+```
+e213184a548871a47fb526f3cba24e2ee2fbbc8129c4ab497ef2ce535130a0a4
+```
+
+Then, to return the principal ID associated with your identity, run the command:
```

Comment:
Shouldn't we also mention that an ICP transfer has to be made to this account in order to create cycles or canisters?